vizro-core/examples/_chart-gallery/pages/time.py

```diff
@@ -65,4 +98,65 @@
 )
 
 connected_scatter = connected_scatter_factory("time")
-pages = [line, column, area, connected_scatter]
+column_and_line = column_and_line_factory("time")
+
+stepped_line = vm.Page(
+    title="Stepped line",
+    path="time/stepped-line",
+    layout=vm.Layout(grid=PAGE_GRID),
+    components=[
+        vm.Card(
+            text="""
+
+            #### What is a stepped line chart?
+
+            A stepped line chart is much like a standard Line chart but, instead of connecting two points with the
+            shortest line, the line forms a series of steps between data points.
+
+            &nbsp;
+
+            #### When should I use it?
+
+            You should use a stepped line chart when you wish to draw attention to changes occurring at specific points.
+            By contrast, a line chart would suggest that changes occur gradually.
+        """
+        ),
+        vm.Graph(
+            figure=px.line(
+                data_frame=gapminder.query("country=='China'"),
```

Comment:
I'm not a big fan of this example because it directly contradicts the above text since life expectancy changes gradually and not in steps.

I'm not sure there's any better examples in `px.data` though so may not be easy to improve this. The stocks data suffers the same problem but somehow looks like nicer to me as a stepped line chart? Maybe just because it's got more points than the gapminder one.

Reply:
Yes, there is not really a good (realistic) example. Let me replace it with the stocks data though 👍